### DISCUSSION TITLE
Copilot for Microsoft Word? - Example video

### DISCUSSION THREAD

**Discussion by titouandk** [community/community#19085] [Copilot Conversations] [answered]
Hello,

Copilot is great 👍 

Will Microsoft ever translate this tool to Word?

It could be a meaningfull argument to opt for the Office subscription model rather than the permanent license, thinking about it 🤔

Edit (add example video):

https://user-images.githubusercontent.com/5413308/174908529-c2b52450-9caa-4ba9-a753-ba8d44fce11b.mov

**Comment by TWiStErRob** [comment]
😮 How do you imagine that? You type the section heading and you get the paragraph?

**Reply by titouandk** [reply]
Maybe could it complete more on the sentence level like so ?
(I chose voluntarily a development subject and gave him a little bit of context) :

https://user-images.githubusercontent.com/5413308/174908529-c2b52450-9caa-4ba9-a753-ba8d44fce11b.mov

Maybe the UI could be adapted in some way to better deal with the MS Word context (Tab used to validate word by word ? Several completions proposed to the user ?)

**Comment by theteladras** [comment, marked as answer]
@titouandk Google docs do have a feature called "Smart Compose", it works somewhat similar, but then again its probably not what you asked for.

However I did notice that Microsoft does have something similar, or at least there is a mention [here](https://support.microsoft.com/en-us/office/editor-text-predictions-in-word-7afcb4f3-4aa2-443a-9b08-125a5d692576?ns=winword&version=90&syslcid=1033&uilcid=1033&appver=zwd900&helpid=173034&ui=en-us&rs=en-us&ad=us#ID0EBBD=Office_365), about it. So might be worth checking it a bit deeper.

**Reply by titouandk** [reply]
Nice, thanks, I did not knew about those 2 options. Sadly for me, both aren't available to me yet :(

**Comment by titouandk** [comment]
They did it 😄

**Reply by TWiStErRob** [reply]
Nice trailer: https://youtu.be/S7xTBa93TX8

https://blogs.microsoft.com/blog/2023/03/16/introducing-microsoft-365-copilot-your-copilot-for-work/

No Clippy this time 😔
<img src="https://media.licdn.com/dms/image/D4D22AQERyUKb6J1ZlA/feedshare-shrink_800/0/1683189180713?e=1687996800&v=beta&t=GQRP0upMnJmP9lbRWxJJDJ5qoJdCcZeawlTQtiaBqw4" width=200px />
